### PR TITLE
Implement GH issue create

### PR DESCRIPTION
**Pull request by kocsenc**

### CODE REVIEW

File: app/controllers/api/github_issues_controller.rb [hunk 1]
```diff
@@ -3,11 +3,13 @@ class Api::GithubIssuesController < ApplicationController
...
   def index
-    @issues = @client.issues.list
+    @issues = @client.issues.list user: 'TeamMartinez', repo: 'YOLO-api', state: 'open'
     render json: @issues
   end
 
   def create
+    # TODO: make user: and repo: params default somewhere and use params
+    @client.issues.create user: 'TeamMartinez', repo: 'YOLO-api', title: params[:title]
     render json: { errors: "not implemented" }
```
Comment (kocsenc) [inline app/controllers/api/github_issues_controller.rb]: I want to use `issues_params` to make `title` required and pass it in to `issues.create`

Comment (jdw6359) [inline app/controllers/api/github_issues_controller.rb]: So it will be `issues.create(issues_params)`

Comment (kocsenc) [inline app/controllers/api/github_issues_controller.rb]: But how will it know about the params coming in from the frontend?

Comment (jdw6359) [inline app/controllers/api/github_issues_controller.rb]: look at other controllers - we need:

```
private
def issues_params
  params.require(:param1, :param2)
end
```

Comment (jdw6359) [inline app/controllers/api/github_issues_controller.rb]: The only thing is that `:require` seems to behave differently than `:permit` so we will need to investigate how to require what we want. I also dont know if `:permit` and `:require` are rails specific things for `params` or if there is general documentation on filtering arrays

Comment (kocsenc) [inline app/controllers/api/github_issues_controller.rb]: Two things
1. `require` can be chained with a `.permit` to encmpass all - And yes they are [rails specific](http://api.rubyonrails.org/classes/ActionController/Parameters.html)
2. What we want is consistent parameters for `user: 'TeamMartinez', repo: 'YOLO-api'`. That doesen't seem like something we can do with `params.require / permit`

**Pull request merged**